apps/showcase/tsconfig.eslint.json

```diff
@@ -1,9 +1,11 @@
 {
   "extends": "./tsconfig.app",
+  "exclude": [],
   "include": [
     ".eslintrc.js",
     "jest.config.js",
     "e2e*playwright/**/*",
-    "testing/setup-jest.ts"
+    "testing/setup-jest.ts",
+    "src/assets/**/*.ts"
```

Comment:
I think a dedicated TSConfig would make more sense for this one (to get intellisense and dedicated libs for example).
Then you will need to register this files as ref in tsconfig.json to get IDE intellisense

Reply:
After discussion, we decided to exclude the asset files from eslint since they will necessarily contain errors (such as unknown relative paths etc), this is because they are combined with a monorepo-template JSON file that is not involved in the eslint process either so it makes sense to exclude the asset files too